scripts/install/mylar.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Mylar Installer for Swizzin
+# Author: Public920
+
+if [[ -f /tmp/.install.lock ]]; then
+    log="/root/logs/install.log"
+else
+    log="/root/logs/swizzin.log"
+fi
+
+user=$(cut -d: -f1 < /root/.master.info)
+codename=$(lsb_release -cs)
+. /etc/swizzin/sources/functions/pyenv
+
+if [[ $codename =~ ("buster"|"focal") ]]; then
+    LIST='git python3-dev virtualenv python3-virtualenv python3-pip'
+else
+    LIST='git build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev curl libbz2-dev'
+fi
+
+apt-get -y update >>"$log" 2>&1
+for depend in $LIST; do
+    apt-get -qq -y install $depend >>"$log" 2>&1 || { echo "ERROR: APT-GET could not install a required package: ${depend}. That's probably not good..."; }
+done
+
+if [[ ! $codename =~ ("buster"|"focal") ]]; then
+    cd /tmp
+    curl -O https://www.python.org/ftp/python/3.8.1/Python-3.8.1.tar.xz
+    tar -xf Python-3.8.1.tar.xz
+    cd Python-3.8.1
+    make
+    make install
+
+    python3_getpip
+fi
+
+python3_venv ${user} mylar
+
+git clone https://github.com/mylar3/mylar3.git /opt/mylar >>"$log" 2>&1
```

Comment:
I would consider pointing this to the releases instead as well. It would be a lot easier to support this app if we can stick to this system, also it will probably mean the updating won't be "broken" after an install. I'm doing something similar in the mango PR.